src/SDKs/DataLake.Store/DataLakeStore.Tests/DataTransferTests/DataLakeStoreClientTests.cs

```diff
@@ -388,7 +388,7 @@ public void DataLakeUploader_ResumeUploadDownloadWithAllMissingFiles()
         /// <summary>
         /// Tests the resume upload when only some segments were uploaded previously with progress tracking enabled
         /// </summary>
-        [Fact]
+        [Fact(Skip = "Randomly failing on CI. Needs investigation from DataLakeStore team")]
```

Comment:
@markcowl I probably won't fix it within the next month. If there is some way to flag this test as a local only test I would like to do that. Otherwise I will need to come up with a more comprehensive plan for fixing the unit tests here to not require local file system access or multiple threads.